src/components/StatusLabel.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  export let level = 'info';
+</script>
+
+<style>
+
+.status-label {
+  font-size: var(--text-015);
+}
+
+.status-label span {
+  display: inline-block;
+  padding: 3px 9px 4px;
+  border-radius: 11.5px;
```

Comment:
This border radius feels like an argument for something like `calc(var(--space-base) * 1.5)` which is `12px` in the current formulation. Is there a reason for `11.5px` vs. `12px`?